kore/src/Kore/Strategies/Goal.hs

```diff
@@ -951,7 +950,7 @@ deriveResults goal results = do
             Result.mapConfigs
                 (flip (configurationDestinationToRule goal) destination)
                 (flip (configurationDestinationToRule goal) destination)
-                (Result.mergeResults results)
+                results
     results' <-
         traverseConfigs (mapRules onePathResults)
```

Comment:
I know this isn't your change, but I just noticed this is still called `onePathResults`.